odpf/guardian/guardian.proto

```diff
@@ -23,7 +23,7 @@ option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_swagger) = {
 };
 
 // WARNING: This is still in active development and can have breaking changes
-service Guardian {
+service GuardianService {
```

Comment:
let's keep it as `Guardian`

Reply:
Sure, but then we need to update it in the Guardian repo as well.

Reply:
yes, will update from an upcoming PR

Reply:
cancelling this suggestion as per this [previous comment](https://github.com/odpf/proton/pull/17#discussion_r670766457)